contrib/pom.xml

```diff
@@ -604,10 +604,10 @@
       <optional>true</optional>
     </dependency>
     <dependency>
-      <!-- Kinesis depends on HttpPatch which requires httpclient 4.3.5 -->
+      <!-- Kinesis depends on HttpPatch which requires httpclient 4.3.6 -->
```

Comment:
The comment is misleading. Kinesis version that contrib optionally depends on depends on 4.3.0 and does not require 4.3.5 or 4.3.6. IMO, it will be better to upgrade Kinesis dependency to at least 1.10.77 that depends on 4.3.6 httpclient.